### PR TITLE
sql: optimize pre_split_regions explanation

### PR DESCRIPTION
**Pull request by 3pointer**
### What is changed, added or deleted? 
Optimize `pre_split_regions` explanation in split-region.md

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by bb7133** [comment]
LGTM, PTAL @winkyao @crazycs520 @zimulala

**Comment by TomShawn** [comment]
@3pointer When the content is ready, please also update v3.1 as well, thanks~

File: v3.0/reference/sql/statements/split-region.md [hunk 1]
```diff
@@ -174,12 +174,13 @@ region4  [("c", "")                    , maxIndexValue               )
 
 ## pre_split_regions
 
-使用带有 `shard_row_id_bits` 的表时，如果希望建表时就做均匀切分 Region，可以考虑配合 `pre_split_regions` 一起使用，用来在建表成功后就开始预均匀切分 `2^(pre_split_regions-1)` 个 Region。
-
+对于带有 `shard_row_id_bits` 的表，如果希望建表时就均匀预切分好 Region，可以使用 `pre_split_regions` 配置。
```
Comment (yikeke) [inline v3.0/reference/sql/statements/split-region.md]:
```suggestion
对于带有 `shard_row_id_bits` 的表，如果希望建表时就均匀预切分好 Region，可以使用 `pre_split_regions` 配置。

```

**Comment by yikeke** [comment]
Please resolve the conflict. Thanks. @3pointer

**Comment by 3pointer** [comment]
This logic already changed

**Pull request closed without merging**